### PR TITLE
Rewrite Should help you start to fix this mess

### PR DESCRIPTION
**Pull request by GuildedThorn**

### CODE REVIEW

**Review by GuildedThorn — COMMENTED** [review]
took a second look over my changes, you should check/test to see if any packet bugs occur, but this should be good

**Comment by onebone** [comment]
I have checked your PR, but I see that most of the codes are not written by you.

**Comment by GuildedThorn** [comment]
I updated the packets to the new pmmp release

**Comment by GuildedThorn** [comment]
And wdym not written by me, i was literally in a vc with jason making these changes in pm's discord

**Comment by GuildedThorn** [comment]
And the only change i took that isnt from u, is a fork and that was to register the boat using mc extended ids

File: Boat-PMMP-master/src/onebone/boat/Main.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+			new ShapelessRecipe(
+				[
+					Item::get(Item::WOODEN_PLANKS, 0, 5),
+					Item::get(Item::WOODEN_SHOVEL, 0, 1)
+				],
+				[Item::get(Item::BOAT, 0, 1)])
+		);
+
+		//Register boat entities
+		Entity::registerEntity(BoatEntity::class, true, ["Boat", "minecraft:boat"]);
```
Comment (GuildedThorn) [inline Boat-PMMP-master/src/onebone/boat/Main.php:32]: The only line i took from a fork of your repo

**Comment by onebone** [comment]
It seems you should make this PR to [here](https://github.com/Blugin/Boat-PMMP), where your code is based off. Requesting directly here without any legal notices and commit logs might cause a violation of copyright law.
Anyway thank you for having attention to my project, have a great day :D

**Pull request closed without merging**